examples/kubernetes/1.11/cilium-crio-ds.yaml

```diff
@@ -221,8 +221,7 @@ spec:
             defaultMode: 420
             optional: true
             secretName: cilium-clustermesh
-      restartPolicy: Always
-      priorityClassName: system-node-critical
+      restartPolicy: Alwaysn      priorityClassName: system-node-critical
```

Comment:
What's going on here?

Reply:
I have had this happen when i ran the commands to generate the `DaemonSet` files on Mac. It needs to be done on Linux :(

Reply:
Fixed